kpops/cli/pipeline_config.py

```diff
@@ -23,7 +23,7 @@ class TopicNameConfig(BaseSettings):
 class PipelineConfig(BaseSettings):
```

Comment:
please run `python gen_schema.py` after changing the descriptions :)

Reply:
done